R/get-path-from-object.R

```diff
@@ -37,13 +37,13 @@ get_model_path <- function(.bbi_object, .check_exists = TRUE) {
 #' @rdname get_path_from_object
 #' @export
 get_model_path.bbi_nonmem_model <- function(.bbi_object, .check_exists = TRUE) {
-  get_model_path_bbi(.bbi_object, .check_exists)
+  get_model_path_nonmem(.bbi_object, .check_exists)
```

Comment:
That the implementations of this method, and the method for `bbi_nonmem_summary`, are identical, suggests that they share an abstract parent class, possibly `bbi_model`?